spacy/attrs.pyx

```diff
@@ -158,8 +158,11 @@ def intify_attr(name):
     stringy_attr (string): Attribute string name. Can also be int (will then be left unchanged)
     RETURNS (int): int representation of the attribute, or None if it couldn't be converted.
     """
+    iob_strs = ("", "I", "O", "B")
```

Comment:
It won't be here in the end, but use the same `Token` property rather than a separate list.

Reply:
OK, sure.

Reply:
OK, I imported `Token` into `attrs.py` but it created a problem while building the project. It looks like it creates a cyclic import and doesn't compile at all. If I remove the import and place a list, it works fine. Here's the error:

```
ImportError while loading conftest '/home/duygu/Templates/spaCy/spacy/tests/conftest.py'.
spacy/__init__.py:14: in <module>
    from . import pipeline  # noqa: F401
spacy/pipeline/__init__.py:1: in <module>
    from .attributeruler import AttributeRuler
spacy/pipeline/attributeruler.py:6: in <module>
    from .pipe import Pipe
spacy/pipeline/pipe.pyx:1: in init spacy.pipeline.pipe
    # cython: infer_types=True, profile=True
spacy/vocab.pyx:1: in init spacy.vocab
    # cython: profile=True
spacy/lexeme.pyx:18: in init spacy.lexeme
    from .attrs import intify_attrs
spacy/attrs.pyx:1: in init spacy.attrs
    from .tokens.token cimport Token
spacy/tokens/__init__.py:1: in <module>
    from .doc import Doc
spacy/tokens/doc.pyx:28: in init spacy.tokens.doc
    from ..attrs import intify_attr, IDS
E   ImportError: cannot import name intify_attr
```

Shall I go back to using a list?